app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -37,7 +37,10 @@ There are full-time and part-time teacher training courses available.
 
 ## What you’ll learn in initial teacher training
 
-What you’ll learn will depend on your course and the qualifications it offers. Some teacher training courses award [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts), while some award QTS with a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce).
+What you’ll learn will depend on your course and the qualifications it offers. You'll train towards one of the following qualifications:
+
+* [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts)
+* [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce) with QTS
```

Comment:
@emmaachesongray We need to say QTS with PGCE, not PGCE with QTS (even though that is slightly more awkward here with the link!)

This is how we refer to it elsewhere round the site - because the key is that QTS is what you **need**, so we need to emphasise it. Find have also updated their terminology to match ours - so it is `QTS with PGCE`

Reply:
```suggestion
* QTS with a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce)
```